product-web-app/controller/JobRolesController.ts

```diff
@@ -15,5 +17,29 @@ export default class JobRolesController {
         res.status(500).send('Error JobRolesController');
       }
     });
+
+    app.get('/admin/job-roles', async (req: Request, res: Response) => {
+      res.render('add-job-roles');
+    });
+
+    app.post('/admin/job-roles', async (req: Request, res: Response) => {
+      const data: AddJobRole = req.body;
+      const jobRoleService = this.jobRoleServiceClass;
+
+      try {
+        if (data.title && data.summary && data.sharepoint_link !== undefined) {
+          const sanitizedData: AddJobRole = {
+            title: xss(data.title),
+            summary: xss(data.summary),
+            sharepoint_link: xss(data.sharepoint_link),
+          };
+          await jobRoleService.createJobRoles(sanitizedData);
+          res.locals.successmessage = 'Success to add job role';
+        }
+      } catch (e) {
+        res.locals.errormessage = 'Failed to add job role';
```

Comment:
it might be useful to add here an error log / console.error ? with details from 'e' of why it failed, what status was returned